src/Authentik/Provider.php

```diff
@@ -56,17 +56,32 @@ protected function getTokenUrl()
     protected function getUserByToken($token)
     {
         $response = $this->getHttpClient()->get(
-            $this->getBaseUrl().'/application/o/userinfo/',
+            $this->getBaseUrl() . '/application/o/userinfo/',
             [
                 RequestOptions::HEADERS => [
-                    'Authorization' => 'Bearer '.$token,
+                    'Authorization' => 'Bearer ' . $token,
                 ],
             ]
         );
 
         return json_decode((string) $response->getBody(), true);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function refreshToken($refreshToken)
```

Comment:
could we type this to string?

Reply:
I guess so. I didn't type it since the rest of the provider isn't.

Should I add the type then?